README.md

```diff
@@ -22,16 +22,12 @@ achieve.
 
 ## Installation
 
-The aim is for you to be able to install this open source research library by
-running:
+This open source research library is abel to be installed by running:
```

Comment:
typo: able

Reply:
> but for a README, spelling is critical to provide a professional appearance

Yup, I completely agree. Thanks Stuart :slightly_smiling_face:. Misspelling, especially in a document like that comes off quite unprofessional.